### PR TITLE
[test](executor)remove murmurHash2_64 in be

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes

1 remove murmurHash2_64 in be, it can not be found in FE.

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

File: be/src/vec/functions/function_hash.h [hunk 1]
```diff
@@ -20,14 +20,6 @@
 
 #pragma once
 
-#include "vec/common/hash_table/hash.h"
```
Comment (yiguolei) [inline be/src/vec/functions/function_hash.h:23]: why not delete this file

Comment (wangbo) [reply be/src/vec/functions/function_hash.h:23]: ```hash.h``` is used by IntHash64Impl. 
```hash.h``` contains int_hash64.

**Comment by wangbo** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**